src/root/components/DrefPdfDocument/index.tsx

```diff
@@ -534,7 +533,7 @@ function DrefPdfDocument(props: Props) {
             {strings.drefExportPreviousOperations}
           </Text>
           <View>
-            <Link src={resolveUrl(window.location.origin, 'https://go.ifrc.org/preparedness#operational-learning')}>
+            <Link src={resolveUrl(window.location.origin, process.env.REACT_APP_OPERATIONAL_LEARNING_PLATFORM_LINK as string)}>
```

Comment:
```suggestion
            <Link src={resolveUrl(window.location.origin, 'preparedness#operational-learning')}>
```
We can simply use this